lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -1,27 +1,26 @@
 """
 axis_artist.py module provides axis-related artists. They are
 
- * axis line
- * tick lines
- * tick labels
- * axis label
- * grid lines
+* axis line
+* tick lines
+* tick labels
+* axis label
+* grid lines
 
 The main artist class is a AxisArtist and a GridlinesCollection. The
 GridlinesCollection is responsible for drawing grid lines and the
 AxisArtist is responsible for all other artists. The AxisArtist class
 has attributes that are associated with each type of artists.
 
- * line : axis line
- * major_ticks : major tick lines
- * major_ticklabels : major tick labels
- * minor_ticks : minor tick lines
- * minor_ticklabels : minor tick labels
- * label : axis label
+* line : axis line
+* major_ticks : major tick lines
+* major_ticklabels : major tick labels
+* minor_ticks : minor tick lines
+* minor_ticklabels : minor tick labels
+* label : axis label
 
-Typically, the AxisArtist associated with a axes will be accessed with
-the *axis* dictionary of the axes, i.e., the AxisArtist for the bottom
-axis is
+Typically, the AxisArtist associated with a axes will be accessed with the
+*axis* dictionary of the axes, i.e., the AxisArtist for the bottom axis is ::
```

Comment:
`is::` (no space)

Reply:
no, I want the space (no colon in the rendered html)